common/ts/helpers/__tests__/request-test.ts

```diff
@@ -0,0 +1,31 @@
+import Endpoint, { EndpointType } from "../../sonarqube/Endpoint";
```

Comment:
You can focus on commonv5 only (it is used only by SQv5 and SonarCloud extensions).